media-video/mpv/mpv-0.11.0-r1.ebuild

```diff
@@ -0,0 +1,278 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_3,3_4} )
+PYTHON_REQ_USE='threads(+)'
+
+WAF_PV='1.8.12'
+
+inherit eutils fdo-mime gnome2-utils pax-utils python-any-r1 waf-utils
+
+DESCRIPTION="Media player based on MPlayer and mplayer2"
+HOMEPAGE="https://mpv.io/"
+SRC_URI="https://waf.io/pub/release/waf-${WAF_PV}"
```

Comment:
Aha. Now, if I understood this correctly, the real issue here is that it is not clearly stated that this line actually fetches a raw executable, rather than an archive of some sort, right?